lib/src/base-styles/_nx-lists.scss

```diff
@@ -133,104 +130,64 @@
   }
 
   .nx-list__item {
-    @include semi-bold();
-
-    color: $nx-text-color-light;
-
     &.selected {
-      background-color: $nx-grey-3;
-      color: $nx-text-color;
+      background-color: #d3e6ff;
       pointer-events: none;
     }
 
-    &:not(.nx-list__item--empty):not(.nx-disabled):not([disabled]) {
+    &:not(.nx-list__item--empty):not(.disabled):not([disabled]),
+    &:not(.nx-list__item--error) {
       &:hover, &:focus {
-        background-color: $nx-color-blue-96;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
-        color: $nx-text-color;
+        background-color: #f1f3fa;
         cursor: pointer;
         outline-style: none; // removes :focus outline
 
         .fa-chevron-right {
```

Comment:
I don't think this style is doing anything, this isn't the right class anymore.  I'm also not seeing where you got the new color value from.  As an aside, in zeplin it looks like the chevron on disabled rows should not be visible.

Reply:
> I don't think this style is doing anything, this isn't the right class anymore. I'm also not seeing where you got the new color value from. As an aside, in zeplin it looks like the chevron on disabled rows should not be visible.

Removed here: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/a4627bcaa98ac3cd0036f79784f12b5a77416963

Reply:
> I don't think this style is doing anything, this isn't the right class anymore. I'm also not seeing where you got the new color value from. As an aside, in zeplin it looks like the chevron on disabled rows should not be visible.

New class for specific targeting and removed in disabled version here: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/51c8de88604a21f5a056174d55cddeb6a2ae0e6f